File: server/server/Dto/ProductDto.cs
namespace server.Dto
{
    public class ProductDto
    {
        public long Id { get; set; }
        public string Name { get; set; }
        public float Price { get; set; }
        public int Amount { get; set; }
        public string Description { get; set; }
        public string Image { get; set; }
        public SellerDto SellerDto { get; set; }
    }
}

File: client/src/dto/ProductDto.js
export default class ProductDto{
    constructor(id, name, price, amount, description, image, sellerDto)
    {
        this.Id = id;
        this.Name = name;
        this.Price = price;
        this.Amount = amount;
        this.Description = description;
        this.Image = image;
        this.SellerDto = sellerDto;
    }
}